pageserver/src/tenant/secondary/downloader.rs

```diff
@@ -827,6 +828,14 @@ impl<'a> TenantDownloader<'a> {
                 }
             }
 
+            // Failpoint for simulating slow remote storage
+            tracing::info!(">>failpoint");
```

Comment:
nit: Doesn't the failpoint get logged by default? In any case, if you want to keep the logs they should be only in test mode.

Reply:
Yes, these were rogue log lines from debugging.  Removed.